applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes.cpp

```diff
@@ -113,7 +113,7 @@ void TwoFluidNavierStokes<TElementData>::CalculateLocalSystem(
                 const unsigned int number_of_gauss_points = gauss_weights.size();
                 array_1d<double, NumNodes> Ncenter;
                 for (unsigned int i = 0; i < NumNodes; i++){
-                    Ncenter[i] = 0.25;
+                    Ncenter[i] = 1.0/NumNodes;
```

Comment:
such changes should usually be in a separate PR since they change the behavior of the element

Reply:
for me it is fine, just pointing it out
this makes it harder to trace errors if things unrelated to a PR are changed